hypha/apply/users/templates/users/activation/email.txt

```diff
@@ -1,8 +1,10 @@
-{% load wagtailadmin_tags %}
+{% load i18n wagtailadmin_tags %}
 {% base_url_setting as base_url %}
-Dear {{ name|default:username }},
+{% name|default:username as user %}
 
-An account on the {{ ORG_LONG_NAME }} web site has been created. Activate your account by clicking this link or copying and pasting it to your browser:
+{% blocktrans %}Dear {{ user }},{% endblocktrans %}
```

Comment:
Want it to look like this and not `{% blocktrans %}Dear {{ name|default:username }},{% endblocktrans %}`.